files/scripts/supervisor-proc-exit-listener

```diff
@@ -35,12 +52,21 @@ def main():
             processname = payload_headers['processname']
             groupname = payload_headers['groupname']
 
-            # If a critical process exited unexpectedly, terminate supervisor
-            if expected == 0 and processname in critical_processes or groupname in critical_processes:
-                MSG_FORMAT_STR = "Process {} exited unxepectedly. Terminating supervisor..."
-                msg = MSG_FORMAT_STR.format(payload_headers['processname'])
-                syslog.syslog(syslog.LOG_INFO, msg)
-                os.kill(os.getppid(), signal.SIGTERM)
+            config_db = swsssdk.ConfigDBConnector()
+            config_db.connect()
+            docker_config = config_db.get_table(CONTAINER_FEATURE_TABLE_NAME)
+            if docker_config and docker_config.has_key(container_name):
```

Comment:
I think we should log if either of these fails, also, like (using the new variable name I requested above:

```
if not container_features_table:
    syslog.syslog(syslog.LOG_ERROR, "Unable to retrieve container features table from Config DB. Exiting...")
    sys.exit(2)

if not container_features_table.has_key(container_name):
    syslog.syslog(syslog.LOG_ERROR, "Unable to retrieve features for container '{}'. Exiting...".format(container_name))
    sys.exit(3)
```

Reply:
Fixed.

Reply:
Hi Joe, I just checked the syslog and we should use syslog.syslog(syslog.LOG_ERR, "...") to write message into syslog file. See this https://docs.python.org/2/library/syslog.html.

Reply:
Yes, correct. I wrote those above example lines off the top of my head. It was not meant to be exact, working code. `LOG_ERR` is the correct constant name, not `LOG_ERROR`.

Reply:
I will update it.